test/xml_builder_test.exs

```diff
@@ -162,8 +162,10 @@ defmodule XmlBuilderTest do
   test "element with attributes" do
     warning =
       capture_io(:stderr, fn ->
-        assert doc(:person, %{occupation: "Developer", city: "Montreal"}) ==
-                 ~s|<?xml version="1.0" encoding="UTF-8"?>\n<person city="Montreal" occupation="Developer"/>|
+        assert doc(:person, %{occupation: "Developer", city: "Montreal"}) in [
+                 ~s|<?xml version="1.0" encoding="UTF-8"?>\n<person city="Montreal" occupation="Developer"/>|,
```

Comment:
The order of these attributes changes between Elixir v1.14 and v1.15, but don't believe it has a significant impact.